apps/dashboard/config/configuration_singleton.rb

```diff
@@ -50,6 +50,7 @@ def boolean_configs
       :disable_bc_shell             => false,
       :cancel_session_enabled       => false,
       :hide_app_version             => false,
+      :relaunch_session_enabled     => false,
```

Comment:
I don't think we want the config. Let's just let folks do it, I can't see why folks _wouldn't_ want it.

Reply:
fixed